docs/contributor-guide/developer-guide/feature-flags.md

```diff
@@ -114,3 +114,20 @@ in the existing tests until you also update them.
 1. Always guard DB write code, even if a path can't be reached unless upstream guarded code was enabled. 
 2. Consider what will happen if your flag is enabled for some time then disabled. Should data views still be accessible? etc.
 3. Always test code with your feature off and on, or other variations.  Our dev environments make it easy to only see the bleeding-edge of features, but real users will be seeing the stable released version of your code.
+
+## Releasing a Feature Behind a Feature Flag
+
+The following process should be used to implement a feature flag, release it in a build, and then remove it once the feature has been used in production.
+
+1. Create a Github issue using the Feature Flag template that covers the creation and deletion of the feature flag that will guard your feature. Ensure it is tied to the appropriate epic so that the issue must be completed before the epic is considered complete.
+2. Create the feature flag in code ([example](https://github.com/civiform/civiform/pull/4435)), update the issue to mention that the feature flag now exists. Enable the feature flag in [dev](https://github.com/civiform/civiform/blob/main/server/conf/application.dev.conf). Disable the flag for [browser tests](https://github.com/civiform/civiform/blob/main/server/conf/application.dev-browser-tests.conf), since the browser tests will turn the flag on and off inside the tests themselves.  For the PR that adds the flag, ensure the release notes section describes what this flag is for.
+3. Write your feature and guard it with the flag.
+4. Pass the flag through the deployment system to allow enabling it in staging ([example](https://github.com/civiform/cloud-deploy-infra/commit/9d17356ff1fa1f3a16c97608cc00cbd4c7c11ffe)).
```

Comment:
JFYI: we'll be able to remove this step soon since we'll be automatically passing flags through the deployment system